packages/venia-concept/src/components/ProductImageCarousel/carousel.js

```diff
@@ -55,7 +55,7 @@ class Carousel extends Component {
             .sort((a, b) => {
                 const aPos = isNaN(a.position) ? 9999 : a.position;
                 const bPos = isNaN(b.position) ? 9999 : b.position;
-                return bPos - aPos;
```

Comment:
@zetlen @jimbo I found in testing that this did the opposite of what I expected.

```js
var numbers = [4, 2, 5, 1, 3];
numbers.sort((a, b) => a - b); // [1, 2, 3, 4, 5]
numbers.sort((a, b) => b-a); // [5, 4, 3, 2, 1]
```